classes/Tools14.php

```diff
@@ -24,2244 +24,2241 @@
  *  International Registered Trademark & Property of PrestaShop SA
  */
 
-if (!defined('_PS_DISPLAY_COMPATIBILITY_WARNING_'))
-    define('_PS_DISPLAY_COMPATIBILITY_WARNING_', false);
+namespace PrestaShop\Module\AutoUpgrade;
 
-if(!class_exists('Tools14',false))
+use Tab;
+
+class Tools14

```

Comment:
Why Tools14?

Reply:
Backward compatibility Tools class for PrestaShop 1.4.
I wanted to keep it to avoid relying on the core Tools.